StellarDotnetSdk.Tests/SorobanServerTest.cs

```diff
@@ -994,6 +1006,117 @@ public async Task TestGetTransactionFails()
         Assert.AreEqual(GetTransactionResponse.GetTransactionStatus.NOT_FOUND, getTransactionResponse.Status);
     }
 
+    [TestMethod]
+    public async Task TestSimulateTransaction()
+    {
+        const string json =
+            "{\n"
```

Comment:
Can you add this to a `*.json` file? Like we do elsewhere in the SDK as well?

Reply:
I was thinking to do that when we work on making the tests in this class true unit tests and moving the existing ones to another integration tests project, that way we can avoid potential file movement (the json file). Do you agree?